projects/canopy/src/styles/mixins.scss

```diff
@@ -234,3 +234,11 @@ $breakpoints: (
   margin-bottom: var(--grid-gutter);
   padding: var(--component-padding);
 }
+
+@mixin lg-element-reset {
```

Comment:
This should be automatically done by `autoprefixer`. Does the button have the `appearance` property at all?

Reply:
Doesn't look like the button has that property. You can just add `appearance: none` in the style for `.lg-btn`. `autoprefixer` will handle the rest. If you are using scss only files in your project make sure you have `autoprefixer` setup.

Reply:
Thanks, removed the vendor specific tags to let autoprefixer do it's job.

All form elements _should_ have an appearence property - from caniuse.com
> The appearance property defines how elements (particularly form controls) appear by default. By setting the value to none the default appearance can be entirely redefined using other CSS properties.

This also appears to be a known and ongoing issue with Safari on iOS.  The following was first asked 10 years ago, and still active in 2019.
[https://stackoverflow.com/questions/5449412/styling-input-buttons-for-ipad-and-iphone](url)

Reply:
Yes we should probably have a look at those form elements and make sure they look alright in iOS 🤔